lightautoml/tasks/losses/cb.py

```diff
@@ -175,6 +191,15 @@ def set_callback_metric(
         if type(metric) is str:
             self.metric = None
             _metric_dict = _cb_metrics_dict[task_name]
+            if task_name == "multi:reg":
+                logger.info2("CatBoost supports only MultiRMSE metric and loss for multi:reg task.")
```

Comment:
misleading message, I suggest "MultiRMSE is set as a metric and loss for CatBoost". As well as in line 199.